README.md

```diff
@@ -33,6 +35,9 @@ cmake ..
 cmake --build .
 cmake --build . --target install
 ```
+On Linux, lagscope is built into /usr/local/bin/lagscope.
+
+On Windows, add 'C:\Program Files (x86)\Lagscope\bin' into PATH environment variables, restart computer, then can run lagscope without specify full path.
```

Comment:
Let the user know that it gets installed to "C:\Program Files (x86)\Lagscope\bin" and not mention anything about PATH variable and restart.